src/services/OrderService.ts

```diff
@@ -237,6 +255,47 @@ const printPicklist = async (picklistId: string): Promise<any> => {
   }
 }
 
+const printPackingSlip = async (shipmentIds: Array<string>): Promise<any> => {
+  try {
+    const maargUrl = store.getters['user/getMaargUrl'];
+    const omstoken = store.getters['user/getUserToken'];
+
+    // Get packing slip from the server
+    const resp = await apiClient({
```

Comment:
As we have configured app as moqui first, so we do not need to get maargUrl separately and we will use the api method for making api calls.